docs/.vitepress/theme/custom.css

```
@@ -1,216 +1,261 @@
-  );
-  --vp-home-hero-image-filter: blur(48px);
+    --vp-c-brand-1: oklch(53% 0.12 218);
+    --vp-c-brand-2: oklch(46% 0.11 218);
+    --vp-c-brand-3: oklch(42% 0.09 218);
+    --vp-c-brand-soft: oklch(84% 0.07 218 / 18%);
+    --vp-c-bg: oklch(98% 0.008 218);
+    --vp-c-bg-alt: oklch(95% 0.014 218);
+    --vp-c-bg-soft: oklch(94% 0.018 218);
+    --vp-c-border: oklch(82% 0.025 218);
+    --vp-c-divider: oklch(86% 0.022 218);
+    --vp-font-family-base: "Avenir Next", Avenir, "Segoe UI", sans-serif;
```

Stylelint (17.10.0)

[error] Expected "Avenir" to be "avenir" (value-keyword-case)

(value-keyword-case)